nio/client/async_client.py

```diff
@@ -602,8 +602,19 @@ def add_response_callback(
 
         Args:
             auth_dict (Dict[str, Any]): The auth dictionary.
+            For example :
```

Comment:
Examples are marked by a `Example` section and the code needs to be marked as such to be syntax highlighted.

You can see how an example is added here https://github.com/poljar/matrix-nio/blob/master/nio/client/async_client.py#L193.